### PR TITLE
fix(jest mock): Add stub for static content

### PR DESCRIPTION
**Pull request by michaelw85**
Fix bug for unit testing component dom that includes static content like a sass file.
Include jest-tranform-stub.

Resolves https://github.com/aurelia/testing/issues/67

### CODE REVIEW

**Comment by michaelw85** [comment]
@huochunpeng Here's a PR introducing jest-transform-stub setup for jest.

**Comment by huochunpeng** [comment]
It's easier to raise a new PR for this on current master.
You will need to touch `skeleton/jest/package.json` and `lib/dependencies.json`.

**Comment by michaelw85** [comment]
@huochunpeng Updated PR.

Added stage component example to the app spec (based on your sample repo). To test this pr add a sass file and require it in the app.html. Without the transform in the package json it will fail.

File: skeleton/jest/package.json [hunk 1]
```diff
@@ -34,6 +35,7 @@
       "json"
     ],
     "transform": {
+      "^.+\\.(css|less|sass|scss|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "jest-transform-stub",
```
Comment (huochunpeng) [inline skeleton/jest/package.json]: Please add ext `styl` as cli supported Stylus.

File: skeleton/scaffold-minimum/test__if_karma_or_jest/unit/app.spec.ext [hunk 2]
```diff
@@ -1,7 +1,32 @@
+import {bootstrap} from 'aurelia-bootstrapper';
 import {App} from '../../src/app';
+import {StageComponent} from 'aurelia-testing';
 
 describe('the app', () => {
```
Comment (huochunpeng) [inline skeleton/scaffold-minimum/test__if_karma_or_jest/unit/app.spec.ext]: We probably should remove this duplicated test now, plus the static import above. Showing StageComponent in demo test code is better and more relevant.

File: skeleton/jest/package.json (hunk 1 above)
Comment (michaelw85) [reply skeleton/jest/package.json]: Added styl to transform regex

File: skeleton/scaffold-minimum/test__if_karma_or_jest/unit/app.spec.ext (hunk 2 above)
Comment (michaelw85) [reply skeleton/scaffold-minimum/test__if_karma_or_jest/unit/app.spec.ext]: Removed test

**Comment by michaelw85** [comment]
@huochunpeng resolved your comments 👍

**Comment by huochunpeng** [comment]
Thx.
@EisenbergEffect this can be merged.

**Pull request merged**